packages/protocol/contracts/governance/Validators.sol

```diff
@@ -845,34 +616,7 @@ contract Validators is IValidators, Ownable, ReentrancyGuard, Initializable, Usi
    * @return Whether a particular address is a registered validator.
    */
   function isValidator(address account) public view returns (bool) {
-    return bytes(validators[account].identifier).length > 0;
-  }
-
-  /**
-   * @notice Returns whether an account meets the requirements to register a validator or group.
-   * @param account The account.
-   * @param noticePeriods An array of notice periods of the Locked Gold commitments
-   *   that cumulatively meet the requirements for validator registration.
-   * @return Whether an account meets the requirements to register a validator or group.
-   */
-  function meetsRegistrationRequirements(
-    address account,
-    uint256[] memory noticePeriods
-  )
-    public
-    view
-    returns (bool)
-  {
-    uint256 lockedValueSum = 0;
-    for (uint256 i = 0; i < noticePeriods.length; i = i.add(1)) {
-      if (noticePeriods[i] >= registrationRequirement.noticePeriod) {
-        lockedValueSum = lockedValueSum.add(getLockedCommitmentValue(account, noticePeriods[i]));
-        if (lockedValueSum >= registrationRequirement.value) {
-          return true;
-        }
-      }
-    }
-    return false;
+    return bytes(validators[account].name).length > 0;
   }
 
   /**
```

Comment:
down here on the `deleteElement` we set `list[lastIndex] = address(0);`
should we  `delete list[lastIndex]` 
Also, locked gold has the same method but does not do this

Reply:
Done